tests/rptest/tests/redpanda_test.py

```diff
@@ -50,11 +55,27 @@ def __init__(self,
             else:
                 num_brokers = 1
 
+        cloud_storage_access_key = test_context.globals.get(
+            self.GLOBAL_S3_ACCESS_KEY, None)
+        cloud_storage_secret_key = test_context.globals.get(
+            self.GLOBAL_S3_SECRET_KEY, None)
+
+        # Enable S3 if AWS creds were given at globals
+        if cloud_storage_access_key is not None and cloud_storage_secret_key is not None:
+            if self.si_settings is not None:
```

Comment:
nit: you can leave out `is not None` in this conditional.

Reply:
I did that cause different reviewers like the `is not None` and some don't.
I just can't win lol

Reply:
I do indeed appreciate the explicit checks for None -- as a general convention, it avoids bugs when the non-None value could also be something falsey, like a 0 integer.

Reply:
LOL..Been there. I think this falls under "authors discretion" then. +1

Reply:
It turns out the existing taskfile is setting these to empty strings rather than Nones, so in this instance we do need to either do a straight truthiness check, or fix the taskfile to do Nones for non-set properties.